enhancements/network/ovn-k-localnet-api.md

```diff
@@ -0,0 +1,514 @@
+---
+title: user-defined-network-segmentation
+authors:
+  - "@ormergi"
+reviewers:
+  - "@maiqueb"
+  - TBD
```

Comment:
start networking this PR in the SDN team, and find an API reviewer from them (I'd suggest @npinaeva ) and an approver.

We also want someone involved from the API team asap. We don't want them getting involved too late.